src/components/App.js

```diff
@@ -44,17 +46,35 @@ class App extends Component<{}, State> {
         this.setState({ notes: _.toArray(result) })
       }
     })
+    getArchive().then(result => {
+      this.setState({ archive: _.toArray(result) })
+    })
     this.setState({ apiVersion: await this.state.mf.apiVersion() })
   }
 
-  updateActiveNote = (note: Note): void => {
+  getNoteFromKey = (key: string): ?Note => {
+    const note = this.state.notes.find(note => note.key === key)
+    return note ? note : null
+  }
+
+  updateActiveNote = (note: Note, save?: boolean): void => {
```

Comment:
Having the `save` boolean as final argument is a bit of a code smell to me. I wonder if there's another way (maybe split into two functions?)